java/src/main/java/com/salesforce/functions/context/SfFnContext.java

```diff
@@ -338,6 +342,32 @@ public void setDeadline(OffsetDateTime deadline) {
   }
 
 
+  public SfFnContext functionNamespace(String functionNamespace) {
+    this.functionNamespace = functionNamespace;
+    return this;
+  }
+
+   /**
+   * Optional Namespace prefix for the functions
+   * @return functionNamespace
+  **/
+  @javax.annotation.Nullable
+  @ApiModelProperty(value = "Optional Namespace prefix for the functions")
+  @JsonProperty(JSON_PROPERTY_FUNCTION_NAMESPACE)
+  @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
+
+  public String getFunctionNamespace() {
+    return functionNamespace;
+  }
```

Comment:
Should we also have an invoking namespace?  Ie, the namespace of the component that invoked the function.  Eg, an install Apex class (package namespace) invokes a global function that is a different namespace.

Reply:
Added it .